lib/capistrano-conditional/deploy.rb

```diff
@@ -25,7 +25,7 @@ def skip_task(name, opts={})
 
     # Need to create stub for method in case called from
     @@deploy_context.send(:task, name) do
-      puts msg.cyan unless opts[:silent]
+      puts msg unless opts[:silent]
```

Comment:
Thanks for the report, but to leave existing functionality for most users let's go with

```ruby
msg = msg.cyan if msg.respond_to?(:cyan)
puts msg unless opts[:silent]
```

or something along those lines.  If you want to make that change, I'd be happy to merge the PR in!